src/components/Card.js

```diff
@@ -33,7 +65,13 @@ export class Card {
       .querySelector(".card__like-button")
       .classList.toggle("card__like-button_active");
   }
-  _handleDeletteButton() {
+
+  _openDeletePopup() {
+    deletePopup.open();
```

Comment:
Needs correction: The card class should not know anything about the confirmation popup

Reply:
Could I get a clarification on this correction, I didn't understand what do you mean about confirmation popup

Reply:
The code for opening the confirmation should be located outside of the Card class